crates/volta-core/src/distro/node.rs

```diff
@@ -153,6 +160,7 @@ impl Distro for NodeDistro {
                 distro: Some(ref hook),
                 ..
             }) => {
+                debug!("Using node.distro hook to determine download URL");
```

Comment:
At this point, we don't actually have the information about where the hook came from (though I have a plan for a refactor that will help support that as well as the `volta list` command). We _do_ however, also have the debug messages about where configs are found (and if multiples are found, that they are being merged) in `hook/mod.rs`

Reply:
Perfect! That makes sense. Its tough to grok the flow of these messages just from the GH review (since files are sorted alpha in GH but obviously that isn't how the actual command line flow works).